modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer-fragment.glsl.js

```diff
@@ -47,12 +48,13 @@ void main(void) {
 
   // Take the global opacity and the alpha from vColor into account for the alpha component
   float a = alpha * vColor.a;
-
+  
+  vec4 color = vec4(vColor.rgb, a);;
   if (a < alphaCutoff) {
-    discard;
+    color = transparentColor;
```

Comment:
Does not account for `opacity`

Reply:
Do you mean `transparentColor` need apply the layer prop `opacity`?

Reply:
It’s not “transparent“, it’s background, and everything the layer renders must apply the opacity.

Reply:
This is fixed now.